Documentation/network/servicemesh/http.rst

```diff
@@ -92,3 +92,51 @@ get back some data, but you will get a 404 error if you make a request to ``<add
       "ISBN-10": "1234567890",
       "ISBN-13": "123-1234567890"
     }
+
+Apply Cilium Network Policy
+===========================
+
+The above example doesn't include any network policy, so all the external traffic is allowed by default.
+Let's apply network policy to lock down the external traffics.
+
+.. literalinclude:: ../../../examples/kubernetes/servicemesh/policy/external-lockdown.yaml
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/policy/external-lockdown.yaml
+
+The same request should be rejected with 503 Service Unavailable status code for any request originating from outside.
```

Comment:
Did you mean `403` here instead of `503`? Also small wording and grammar suggestion:

```suggestion
The same request from the previous example should now be rejected with a 403 Service Unavailable status code:
```

Another option to make the statement more generic:

```suggestion
With this policy applied, any request originating from outside the cluster will be rejected with a 403 Service Unavailable status code:
```